app/src/main/java/ch/epfl/sdp/login/LoginFormActivity.java

```diff
@@ -31,20 +34,23 @@ protected void onCreate(Bundle savedInstanceState) {
 
         // If the user has already logged in, go to MainActivity directly
         if (authenticationAPI.getCurrentUserEmail() != null) {
-            startActivity(new Intent(LoginFormActivity.this, MainActivity.class));
+            startActivity(new Intent(LoginFormActivity.this, MainMenuActivity.class));
```

Comment:
better use findbyId

Reply:
I use findById, and this is your doings ^^